analysis/report_templates/default.html

```diff
@@ -228,6 +233,32 @@ <h5 class="center-align">Mann-Whitney U test</h4>
                             </div>
                         </div> <!-- row -->
 
+                        {% if differential_graphs %}
+                        <div class="row">
+                            <div class="col s6 offset-s3">
+                                <h5 class="center-align">Ranking by rare region covered</h4>
+                                    <img class="responsive-img materialboxed"
+                                         src="{{ benchmark.rare_region_ranking_plot }}">
+                                    The table shows the coverage rank of each fuzzer
+                                    on the rare regions for this benchmark. The rare
+                                    region is defined as only being hit by less than
+                                    threshold(default=20%) of all fuzzers.
+                            </div>
+                        </div> <!-- row -->
+
+                        <div class="row">
```

Comment:
Do you happen to have a demo report with these correlation tables? It sounds very interesting! Just wondering if it should be in a "collapsible box"

Reply:
I have attached a demo report in the description!

Reply:
Putting it in the collapsible box is weird because the box says it's only for "Sample statistics and statistical significance" which is not true anymore. Maybe we should change that text, add a new box or not put these in a box at all.

Reply:
It is in a dedicated coverage box @jonathanmetzman 
"Coverage reports for each fuzzer on this benchmark"
http://commondatastorage.googleapis.com/fuzzbench-reports/aarya-2020-08-10/index.html

Reply:
The coverage reports are in their own box. THe differential coverage graphs are not.

Reply:
Yes, the diff cov plots do not belong in the "Sample statistics and statistical significance" box, they should have a separate box (if we want to put them into a collapsible box). 

In general, i think we should put stuff in collapsible boxes that take a lot of space but not everyone wants to see them. On this note, currently we have a "Coverage reports for each fuzzer on this benchmark", which is just a list of links to each fuzzer. Since this list takes minimal space, I think this doesn't necessary need to be in a collapsible box.

Reply:
Yeah makes sense. I'll put the differential graphs to a separate collapsible box. But I think the links to coverage report should still remain in their own collapsible box, since a full experiment may get several lines filled with bottoms and that may look ugly if not in a collapsible box.